src/Umbraco.Web/Security/BackOfficeSignInManager.cs

```diff
@@ -269,12 +269,12 @@ public async Task<string> GetVerifiedUserNameAsync()
         /// This is implemented because we cannot override GetVerifiedUserIdAsync and instead we have to shadow it
         /// so due to this and because we are using an INT as the TKey and not an object, it can never be null. Adding to that
         /// the default(int) value returned by the base class is always a valid user (i.e. the admin) so we just have to duplicate
-        /// all of this code to check for -1 instead.
+        /// all of this code to check for SuperUserId-1 instead.
         /// </remarks>
         public override async Task<SignInStatus> TwoFactorSignInAsync(string provider, string code, bool isPersistent, bool rememberBrowser)
         {
             var userId = await GetVerifiedUserIdAsync();
-            if (userId == -1)
+            if (userId == Constants.Security.SuperUserId - 1)
```

Comment:
If we return 0, seems it would be easier to check against 0 instead of this math calc. Would be good to just store a `const` on this class like `private const int NoUser = 0;`